pkgs/development/compilers/openjdk/darwin/default.nix

```
@@ -7,10 +7,11 @@ let
   };
 
   jdk = stdenv.mkDerivation rec {
-    name = "zulu15.28.51-ca-jdk15.0.1";
+    pname = "zulu15.28.51-ca-jdk";
+    version = "15.0.1";
```

```suggestion
    pname = "zulu";
    version = "15.28.51-ca-jdk15.0.1";
```

maybe? but I am not sure.

Also just found the same bug.

---

What bug? jdk seems to work now
```
nix-repl> jdk.version
"15.0.1"
```